Code/RDGeneral/types.cpp

```diff
@@ -32,6 +32,7 @@ const std::string _BondsPotentialStereo = "_BondsPotentialStereo";
 const std::string _ChiralAtomRank = "_chiralAtomRank";
 const std::string _CIPCode = "_CIPCode";
 const std::string _CIPRank = "_CIPRank";
+const std::string _CanonicalRankingNumber = "CanonicalRankingNumber";
```

Comment:
it's probably better to prefix this with an underscore:
```suggestion
const std::string _CanonicalRankingNumber = "_CanonicalRankingNumber";
```
it's not a hard rule, but we use that for properties that are for "internal" use and not necessarily intended to be used elsewhere

Reply:
I saw that and did it.